### PR TITLE
Custom components template.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: components/settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+include(":VideoPlayer")
```
Comment (igordmn) [inline components/settings.gradle.kts:1]: Maybe we keep only two modules? The purpose of ":VideoPlayer" is unclear.

":VideoPlayer:common" will be for the main library which we will publish in the maven.
And ":VideoPlayer:desktop" is for examples?

Comment (olonho) [reply components/settings.gradle.kts]: Yep, indeed, fixed.

Comment (igordmn) [reply components/settings.gradle.kts]: We can also remove file `components/VideoPlayer/build.gradle.kts`

File: components/VideoPlayer/common/src/commonMain/kotlin/org/jetbrains/compose/videoplayer/VideoPlayer.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.compose.videoplayer
+
+import androidx.compose.runtime.Composable
+
+@Composable
+fun VideoPlayer(url: String, width: Int, height: Int) {
+    VideoPlayerImpl(url, width, height)
+}
+
+expect fun VideoPlayerImpl(url: String, width: Int, height: Int)
```
Comment (igordmn) [inline components/VideoPlayer/common/src/commonMain/kotlin/org/jetbrains/compose/videoplayer/VideoPlayer.kt:10]: internal?

Comment (olonho) [reply components/VideoPlayer/common/src/commonMain/kotlin/org/jetbrains/compose/videoplayer/VideoPlayer.kt]: Yep, think alike, already fixed that in update :).

**Pull request merged**